examples/cms-wordpress/.env.example

```diff
@@ -0,0 +1,5 @@
+NEXT_EXAMPLE_CMS_WORDPRESS_API_URL=
```

Comment:
Let's name this file `.env.local.example`.

Reply:
Done!